src/components/SideNavbar/SideNavMenu.js

```diff
@@ -0,0 +1,313 @@
+  height: 21px;
+  margin: 22px 112px 15.5px 8px;
+  font-size: 11px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+  color: #222;
+}
+`;
+
+const HistoryList = styled.div`
+  width: 270px;
+  height: 76px;
+  border-top: 1px solid #eee;
+  border-bottom: 1px solid #eee;
+  align-items: center;
+  font-size: 14px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+`;
+
+const HistoryLink = styled.div`
+  width: 270px;
+  height: 76px;
+  margin: 27.5px 0;
+  align-items: center;
+  font-size: 14px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+`;
+const HistoryText = styled.span`
+  width: 52px;
+  height: 21px;
+  margin: 27.5px 194px 27.5px 24px;
+
+  font-size: 13px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+  color: #222;
+`;
+
+const ChatList = styled.div`
+  width: 270px;
+  height: 76px;
+  display: flex;
+  align-items: center;
+  font-size: 14px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+`;
+
+const ChatLink = styled.div`
+  width: 270px;
+  height: 44px;
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+`;
+
+const ChatText = styled.div`
+  width: 39px;
+  height: 21px;
+  padding: 12px 207px 11px 24px;
+`;
+
+const SetList = styled.div`
+  width: 270px;
+  height: 92px;
+  padding: 365px 0 0 0;
+  display: flex;
+  flex-direction: column;
+  font-size: 14px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+`;
+const Set = styled.div`
+  width: 270px;
+  height: 44px;
+  padding: 0 0 4px 0;
+  display: flex;
+  align-items: center;
+  font-size: 14px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+`;
+
+const SetText = styled.div`
+  width: 30px;
+  height: 24px;
+  padding: 12px 208px 8px 12px;
+`;
+
+const Contact = styled.div`
+  width: 270px;
+  height: 44px;
+  display: flex;
+  align-items: center;
+  font-size: 14px;
+  font-weight: 500;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+`;
+
+const ContactText = styled.div`
+  width: 59px;
+  height: 24px;
+  padding: 32px 179px 12px 12px;
+`;
+
+const Extension = () => {
+  return (
+    <SideMenuBackground>
+      <SideMenuContainer>
+        <Logo src="/asset/Group 1583.png" alt="Logo" />
```

Comment:
`Group 1583` 의 의미가 뭐죠?? 딱 봤을 때 어떤 의미인지 이해할 수 있나요?
알아보기 쉬운 명칭으로 수정해주세요.
```suggestion
        <Logo src="/asset/logo.png" alt="Logo" />
```

Reply:
이미지 파일 이름 수정하겠습니다